examples/scenes/src/svg.rs

```diff
@@ -94,16 +96,43 @@ pub fn svg_function_of<R: AsRef<str>>(
     contents: impl FnOnce() -> R + Send + 'static,
 ) -> impl FnMut(&mut Scene, &mut SceneParams) {
     fn render_svg_contents(name: &str, contents: &str) -> (Scene, Vec2) {
+        use crate::pico_svg::*;
         let start = Instant::now();
-        let svg = usvg::Tree::from_str(contents, &usvg::Options::default())
-            .unwrap_or_else(|e| panic!("failed to parse svg file {name}: {e}"));
-        eprintln!("Parsed svg {name} in {:?}", start.elapsed());
-        let start = Instant::now();
-        let mut new_scene = Scene::new();
-        vello_svg::render_tree(&mut new_scene, &svg);
-        let resolution = Vec2::new(svg.size.width() as f64, svg.size.height() as f64);
-        eprintln!("Encoded svg {name} in {:?}", start.elapsed());
-        (new_scene, resolution)
+        match PicoSvg::load(contents, 1.0) {
+            std::result::Result::Ok(PicoSvg { items, size }) => {
```

Comment:
Why use the fully qualified path here?

Reply:
Conflicting `Ok` and `Result` in the namespace.

Reply:
Hmm, that's annoying.

Maybe it should it use the same fix as in #516 , where the helper  type is defined as:
```rust
type Result<T, E = {default}> = Result<T, E>;
```
?

Reply:
I'm too dumb for this, sorry.

Reply:
Hmm, I'm surprised that this fails. I'll need to dig into it more myself.

I don't really understand how it could fail